rosa_getting_started/rosa-getting-started-workflow.adoc

```diff
@@ -1,22 +1,29 @@
+[id="rosa-understanding-the-deployment-workflow"]
+= Understanding the ROSA deployment workflow
 include::modules/attributes-openshift-dedicated.adoc[]
-
-[id="rosa-getting-started-workflow"]
-= Getting started workflow
-:context: rosa-getting-started-workflow
+:context: rosa-understanding-the-deployment-workflow
 
 toc::[]
 
-Follow this workflow to set up and access {product-title} (ROSA) clusters:
+Before you create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you must complete the AWS prerequisites, verify that the required AWS service quotas are available, and set up your environment.
+
+This document provides an overview of the ROSA with STS deployment workflow stages and references to detailed resources for each stage.
```

Comment:
I think either: 

- "...references detailed..."
- "...refers to detailed..."

Reply:
Thanks for highlighting this. I have updated this to "refers to detailed".